constrainedfilefield/static/constrainedfilefield/js/file_checker.js

```diff
@@ -1,3 +1,20 @@
+function humanReadableSize(bytes) {
+    units = ['Bytes', 'KiB', 'MiB', 'GiB', 'TiB', 'PiB', 'EiB', 'ZiB', 'YiB']
+    var initialbytes = bytes
+
+    var i = 0
+    while (bytes >= 1024) {
```

Comment:
Please use existing operators instead, like `i = Math.floor(bytes/1024)` or `i = ~~(bytes/1024) ` (preferred, as apparently faster, according to https://stackoverflow.com/a/17218003/2553539).
Then you can still get the number of bytes like `bytes = initialbytes/(1024*i)`